### DISCUSSION TITLE
Feedback - v220419 - Reference

### DISCUSSION THREAD

**Discussion by edzob** [JRBliekendaal/master-thesis#142] [General]
It is good to check your references (bibliography) section and your bib files.

## type of citation
For example:
```
Aghina, W., Ahlback, K., Smet, A. D., Lackey, G., Lurie, M., Murarka, M., & Handscomb, C. (2018, January). The five trademarks of agile organizations (research rep.). McKinsey & Company. https://www.mckinsey.com/business- functions/
people - and - organizational - performance/our -insights/the - five - trademarks - of -
agile-organizations#
```
is not referenced as a website.
https://apastyle.apa.org/style-grammar-guidelines/references/examples/webpage-website-references

and steven is referenced as a website:
```
De Haes, S., Van Grembergen, W., Joshi, A., & Huygh, T. (2020). Enterprise governance
of information technology: Achieving alignment and value in digital organizations
[OCLC: 1121274337]. Retrieved November 26, 2020, from https://doi.org/10.
1007/978-3-030-25918-1
```

## field marking
And some authors have their first name or first letter as part of their lastname in the references.
I think this is because of extra { }  in the name field.
for example in the reference of
```
Botjes, Edzo A., van den Berg, Martin, van Gils, Bas & Mulder, Hans. (2021, August).
Attributes relevant to antifragile organizations
```

and of 
`"Graves, T. (2009, June 16). Enterprise architecture: A pocket guide. IT Governance Publishing (ITGP)." `

## Field defintions
`Greefhorst, D., & Proper, E. (2011). Architecture principles: The cornerstones of enterprise architecture [OCLC: ocn729345210]. Springer.
`
## Capital usage
```
Guggenberger, T., Möller, F., Haarhaus, T., Gür, I., & Otto, B. (2020). ECOSYSTEM TYPES IN INFORMATION SYSTEMS. https://www.researchgate.net/
publication/341188637_ECOSYSTEM_TYPES_IN_INFORMATION_SYSTEMS
```

**Comment by JRBliekendaal** [comment]
Aghina, W., Ahlback, K., Smet, A. D., Lackey, G., Lurie, M., Murarka, M., & Handscomb, C. (2018, January). The five trademarks of agile organizations (research rep.). McKinsey & Company. https://www.mckinsey.com/business- functions/
people - and - organizational - performance/our -insights/the - five - trademarks - of -
agile-organizations#

Corrected to @Online

----------------------------------------------

De Haes, S., Van Grembergen, W., Joshi, A., & Huygh, T. (2020). Enterprise governance
of information technology: Achieving alignment and value in digital organizations
[OCLC: 1121274337]. Retrieved November 26, 2020, from https://doi.org/10.
1007/978-3-030-25918-1

Corrected bib. Was @Book but retrieved from (is an springerlink) was there. Rewrote the bib and corrected the sort of Steven his last name from Haes to De Haes. De is part of the family name.

----------------------------------------------

Botjes, Edzo A., van den Berg, Martin, van Gils, Bas & Mulder, Hans. (2021, August).
Attributes relevant to antifragile organizations

Corrected the reference auto generation with prefixes.

----------------------------------------------

"Graves, T. (2009, June 16). Enterprise architecture: A pocket guide. IT Governance Publishing (ITGP)."

Corrected the date field. (Year field is grandfathered).

----------------------------------------------

Greefhorst, D., & Proper, E. (2011). Architecture principles: The cornerstones of enterprise architecture [OCLC: ocn729345210]. Springer.

Corrected in BIB

----------------------------------------------

Guggenberger, T., Möller, F., Haarhaus, T., Gür, I., & Otto, B. (2020). ECOSYSTEM TYPES IN INFORMATION SYSTEMS. https://www.researchgate.net/
publication/341188637_ECOSYSTEM_TYPES_IN_INFORMATION_SYSTEMS

Placed the formal URL without caps.

----------------------------------------------
